### PR TITLE
Refactoring to allow ignore file in all trivy scan

### PR DESCRIPTION
**Pull request by louisledev**
follow-up on DEVOPS-223

### CODE REVIEW

File: scanBinaries.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 parameters: 
   artifactName:  # name of the artifact on Azure DevOps that contains the Zip file                         
   zipfile: # name or pattern of the zipfile to be extracted and scanned
+  trivyIgnoreFile: '' # path to the trivy ignore file
```
Comment (cknaap) [inline scanBinaries.yml:4]: Why do the parameters in this file not have the type, default and displayName specified, like in scanCompiledArtifacts.yml and in scanDockerImage.yml?

File: scanWithRetryTask.yml [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
 steps:
 - script: |
     retries=10
     count=0
-    echo "Executing '${{ parameters.command }}' with retries..."
+    if [ -n "${{ parameters.trivyIgnoreFile }}" ]; then
+      mountArgs="-v ${{ parameters.trivyIgnoreFile }}:/tmp/trivyignore"
+      echo "Using ignore file at ${{ parameters.trivyIgnoreFile }}"
+    else 
+      mountArgs="-v /tmp:/tmp"
```
Comment (cknaap) [inline scanWithRetryTask.yml:16]: why do you need to mount `/tmp` here?

**Review by cknaap — CHANGES_REQUESTED** [review]

File: scanWithRetryTask.yml (hunk 2 above)
Comment (louisledev) [reply scanWithRetryTask.yml:16]: Well, no bertter answer that this is how this was before. I looked into the doc, and could not see any reason, so removed it, will see if this works.

File: scanBinaries.yml (hunk 1 above)
Comment (louisledev) [reply scanBinaries.yml:4]: No better answer that usually we don't provide the name, type, display and default value in our templates, but this seems to be a good practice, so let's start.

**Review by cknaap — APPROVED** [review]

**Pull request merged**